### DISCUSSION TITLE
Additional timezone settings in Display Conditions

### DISCUSSION THREAD

**Discussion by kmgardner** [elementor/elementor#28559] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I have a production site where I would like to use Display Conditions based on date & time. Currently, the only options for selecting a date and time are based on either the server timezone or visitor timezone. My server timezone is set to UTC, but the timezone in the Wordpress application settings is set to Central as this is where the business is located. I would like the displayed items to run against the Central timezone. I would rather not change the server timezone because it works just fine for everything else related to the site and there could be unintended consequences that go along with changing the timezone on a production server. This means I have to account for the -5 hour difference when setting items to display at a specific time. For example, if I want something to display on 9/10/24 at 8am Central time, I have set it to 9/10/24 1pm using Server Time (UTC). 

### Describe the solution you'd like.

It would be nice if there was an option to select "Wordpress timezone" in the Display Conditions and this would use the timezone option from Settings > General in the Wordpress admin.

### Describe alternatives you've considered.

This issue could be solved by changing the server's timezone, but doing so can have unintended consequences for an already established production site.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by denisgomesfranco** [comment]
September 2025 and here I am researching this problem and stubling upon this request that is still open.

And the funniest thing is that Elementor (now at v3.31) has a "server time" and a "visitor time" option but the latter is disabled. That is, they took the time to implement something that doesn't work instead of something ("Wordpress time") that would be actually useful.
